src/content/apis/sports/API-FOOTBALL.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
This API already exists under `sports API`

Reply:
Make sure to check the website and you will see it there 🙏 
<img width="988" alt="Screen Shot 2022-10-25 at 9 20 53 PM" src="https://user-images.githubusercontent.com/23402178/197818200-806fd807-8c61-4eff-92eb-6d06776e8ffb.png">

Reply:
Pardon my mistake 😶